keras_cv/layers/regularization/drop_path_test.py

```diff
@@ -46,3 +46,24 @@ def test_input_gets_partially_zeroed_out_in_train_mode(self):
         non_zeros_outputs = tf.math.count_nonzero(outputs, dtype=tf.int32)
 
         self.assertGreaterEqual(non_zeros_inputs, non_zeros_outputs)
+
+    def test_strict_input_gets_partially_zeroed_out_in_train_mode(self):
+        layer = DropPath(drop_rate=0.5)
+        inputs = tf.random.uniform(self.FEATURE_SHAPE)
+
+        total_non_zero_inputs = 0
+        total_non_zero_outputs = 0
+
+        for _ in range(5000):
+            outputs = layer(inputs, training=True)
```

Comment:
What’s the runtime of this?  5k iterations is quite large.

Reply:
The runtime for this test only is 11.38 seconds.

Reply:
When I was testing this, 5k iterations was the first mark where I got the test to pass on multiple runs. However when I set the seed to 42, the same test passes in 1000 iterations and takes 7.11 seconds. Could you please suggest what should be done in this case?

Reply:
Correction to my last comment, there was a small mistake. The `FEATURE_SHAPE` was set to `(1, 14, 14, 256)` which is a slightly unlikely scenario. I changed it to `(16, 14, 14, 256)` and ran for 200 iterations (with seed), which takes 6.11 seconds.